src/main/java/com/google/sps/servlets/userServlet.java

```diff
@@ -15,11 +15,11 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import com.google.sps.MockUserData;
 
 @WebServlet("/userinformation")
-public class userServlet extends HttpServlet {
+public class UserServlet extends HttpServlet {
 
+  //retrieve user's entity based on user's name
```

Comment:
It'll be great if you can follow the JavaDocs style for comments! http://www.drjava.org/docs/user/ch10.html
The rest of us have been keeping to that style for consistency.

Reply:
Oh also pls remember to rename the file itself to UserServlet.java. Things will break if the file name isn't the same as the (public) class name.